pkg/tls/verifications.go

```diff
@@ -0,0 +1,94 @@
+package tls
+
+import (
+	"errors"
+	"reflect"
+	"strings"
+
+	"github.com/absmach/mproxy/pkg/tls/verifier"
+	"github.com/absmach/mproxy/pkg/tls/verifier/crl"
+	"github.com/absmach/mproxy/pkg/tls/verifier/ocsp"
+	"github.com/caarlos0/env/v10"
+)
+
+var ErrInvalidClientValidation = errors.New("invalid client validation method")
+
+type verification int
+
+const (
+	OCSP verification = iota + 1
+	CRL
+)
+
+func NewVerification(opts env.Options) ([]verifier.Verifier, error) {
+	if opts.FuncMap == nil {
+		opts.FuncMap = make(map[reflect.Type]env.ParserFunc)
+	}
+	opts.FuncMap[reflect.TypeOf(make([]verification, 0))] = envParseSliceValidate
+	opts.FuncMap[reflect.TypeOf(new(verification))] = envParseValidation
+
+	var c struct {
+		Verifications []verification `env:"_CERT_VERIFICATION_METHODS"             envDefault:""`
+	}
```

Comment:
mTLS verification method is not loading properly from .env 

```suggestion
		Verifications []verification `env:"CLIENT_CERT_VERIFICATION_METHODS"             envDefault:""`
```

and also 
Please update .env to 
`MPROXY_MQTT_WITH_MTLS_CLIENT_CERT_VERIFICATION_METHODS=ocsp`
`MPROXY_MQTT_WS_WITH_MTLS_CLIENT_CERT_VERIFICATION_METHODS=ocsp`
`MPROXY_HTTP_WITH_MTLS_CLIENT_CERT_VERIFICATION_METHODS=ocsp`